platform/ios/PolyPodApp/PolyPod/AppDelegate.swift

```diff
@@ -1,17 +1,25 @@
 import BackgroundTasks
 import UIKit
 import CoreData
+import PolyPodCoreSwift
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
     private static let updateNotificationCheckIdentifier = "coop.polypoly.polypod.updateNotificationCheck"
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
-        
         Log.bootstrap()
         Log.info("Application initialized")
         
+        switch Core.instance.bootstrap(languageCode: Language.current) {
+        case .success:
+            Log.info("Core bootstraped!!!")
```

Comment:
Nit: If that message is to stay, it could be less excited I guess :D

Reply:
I would trade exclamation marks only for 🎆 :D. 

I will leave only one `!` then.

Reply:
You got yourself a deal!